orga/app/styles/pages/authenticated/campaigns/details/participants/participant-results.scss

```diff
@@ -0,0 +1,144 @@
+.participant-results {
+  width: 100%;
+
+  &__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0;
+  }
+
+  &__information {
+    margin-bottom: 10px;
+
+  }
+
+  &__result {
+    padding: 10px;
+  }
+}
+
+.participant-results-header {
+  &__title {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.participant-results-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.participant-results-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 2%;
+
+  &--large {
+    padding: 30px 0;
+  }
+
+  &--wide {
+    width: 100%;
+  }
+
+  &--first-part {
+    border-right: 1px solid $alto;
+    width: 60%;
+    margin: 0 0 0 3%;
+    flex-shrink: 1;
+  }
+
+  &--second-part {
+    display: flex;
+    align-items: flex-start;
+    padding: 0 30px;
+    justify-content: space-around;
+    flex-shrink: 0;
+    flex-grow: 1;
+
+    .participant-results-content--large {
+      flex-shrink: 0;
+    }
+  }
+
+  &--multiple {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 50%;
+    padding: 0 30px;
+  }
```

Comment:
A quoi correspond multiple ?

Reply:
pareil, à une partie de la page, j'ai pas mieux comme nom non plus... preneuse d'inputs

Reply:
Sur mon-pix, on a `.rounded-panel-one-line-header` et `.rounded-panel-header-with-components`

Reply:
En étant très littéral, peut-être un .domains-and-skills-counts-container

Reply:
on a opté pour __summary